src/transformers/modeling_outputs.py

```diff
@@ -245,6 +245,47 @@ class BaseModelOutputWithPastAndCrossAttentions(ModelOutput):
     cross_attentions: Optional[Tuple[torch.FloatTensor]] = None
 
 
+@dataclass
+class BaseModelOutputWithNoAttention(ModelOutput):
+    """
+    Base class for model's outputs, with potential hidden states.
+
+    Args:
+        last_hidden_state (`torch.FloatTensor` of shape `(batch_size, sequence_length, hidden_size)`):
```

Comment:
Last hidden states have shape (B, C, H, W) right?

Reply:
Correct, nice catch

Reply:
should we call it `ImageBaseModel...` or something that suggests we are talking about images? @sgugger

Reply:
Resolved since we are going to add these outputs in this PR